requirements/base.in

```diff
@@ -31,6 +31,8 @@ dns-lexicon==3.3.13
 djangorestframework-simplejwt==4.4.0
 docutils==0.14
 drf-yasg
+elasticsearch>=6.0.0,<7.0.0
+elasticsearch-dsl>=6.0.0,<7.0.0
```

Comment:
Are these required on the Ocim server where the command will be run? The elasticsearch query script will only run on the remote host right?

Reply:
Yea, I think that's likely true